### PR TITLE
add scientific calcultor

### LINKED ISSUE #125: I want to add a calculator with more than just basic functions using C language
Please include a small description or summary of the issue here.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Additional context**

**Languages**
- [ ] Python
- [ ] Java
- [x] C
- [ ] C++

I will be adding a calculator with sqrt, log, power func along with basic operations using C language
[c calculator for github.txt](https://github.com/kaustubhgupta/Hacktoberfest2020_/files/5319449/c.calculator.for.github.txt)

### PR DESCRIPTION
**Pull request by ishanbagchi**
# add a Scientific Calculator

Add a command-line scientific calculator that does a little more than classical addition, subtraction, multiplication, and division. It calculates the sin, cos, tan, and much more like this.

Fixes: #125

## Add a scientific calculator in Python language and an instruction.md file

Please delete options that are not relevant.

-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Python/Calculator/INSTRUCTIONS.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- click enter button 
+- enter the second number
+- click enter button
+
+
+## FOR OPERATORS ENTER THE FOLLOWING
+
+you can use upper or lower case for operations    
+
+Operators|Functions
```
Comment (kaustubhgupta) [inline Python/Calculator/INSTRUCTIONS.md]: The readme needs proper alignment right now it's all messed up:
image

File: Python/Calculator/calculator.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+'''
```
Comment (kaustubhgupta) [inline Python/Calculator/calculator.py]: Remove this docstring as you have created a separated file for this

File: Python/Calculator/calculator.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+for FACTORIAL and TRIG you need to put zero (or any number) as the first number, then the sign and last the number you want to calculate. The first number (or the zero) will not be used for calculations but it will prevent an error
+
+
+THE CODE
+'''
+import math
+from builtins import print
+
+print ("Please read instructions in the code\nor reach https://github.com/kaustubhgupta/Hacktoberfest2020_ for help \n*************************")
```
Comment (kaustubhgupta) [inline Python/Calculator/calculator.py]: Remove this print statement

File: Python/Calculator/calculator.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+firstNumber = float(input())
```
Comment (kaustubhgupta) [inline Python/Calculator/calculator.py]: While taking input, print something

File: Python/Calculator/calculator.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+op = input().lower()
+secondNumber = float(input())
+
+# functions of the calculator
+if op == "+":
```
Comment (kaustubhgupta) [inline Python/Calculator/calculator.py:9]: Implement a class-based code, move all this code to a class, and create separate functions for each operation. If the operator requires only one operand then don't take the second number input and directly output the result.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Please correct the following things

File: Python/Calculator/calculator.py [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+elif op == "tan":
+    print ("tan(", secondNumber, ")=", math.tan(secondNumber ))
+elif op == "pie" or op == "pi":
+    print ("Pie =", math.pi)
+elif op == "e":
+    print = ("E =", math.e)
+elif op == "ln":
+    print ("ln(", secondNumber , ")= ", math.log(secondNumber))
+else:
+    print ("incorrect operator")
```
Comment (kaustubhgupta) [inline Python/Calculator/calculator.py:63]: Make the entry of this program in Python directory readme

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by ishanbagchi** [comment]
@kaustubhgupta sorry for the inconvenience. I'll correct it right away.

File: Python/Calculator/calculator.py [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    firstNumber = float(input("Enter the first number: "))
+    secondNumber = float(input("Enter the second number: "))
+    print (firstNumber, "root", secondNumber, "=", secondNumber ** (1 / firstNumber) )
+elif op == "%":
+    firstNumber = float(input("Enter the first number: "))
+    secondNumber = float(input("Enter the second number: "))
+    print(firstNumber, "%", secondNumber, "=", firstNumber % secondNumber )
+elif op == "!":
+    secondNumber = float(input("Enter the number: "))
+    theNumber = firstNumber = secondNumber
```
Comment (kaustubhgupta) [inline Python/Calculator/calculator.py:39]: What is this?

File: Python/Calculator/calculator.py (hunk 5 above)
Comment (kaustubhgupta) [reply Python/Calculator/calculator.py:9]: What about the class code?

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by ishanbagchi** [comment]
@kaustubhgupta for this simple calculator program, if else is comparatively easy. So I thought of do it in this manner.

**Comment by kaustubhgupta** [comment]
@ishanbagchi okay don't implement class code but fix the error I pointed.

**Comment by ishanbagchi** [comment]
> @ishanbagchi okay don't implement class code but fix the error I pointed. 

Which error? I solved the error in INSTRUCTIONS.md, added the link in the language README.me, removed the print statement with the link. Anything else was there? 👀

File: Python/Calculator/calculator.py (hunk 7 above)
Comment (kaustubhgupta) [reply Python/Calculator/calculator.py:39]: The line 39 doesn't seem right

Comment (ishanbagchi) [reply Python/Calculator/calculator.py:39]: This is correct.

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**